paper-slider.html

```diff
@@ -90,6 +90,10 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
`:host-context()` will no longer be support in V1 - see https://github.com/Polymer/polymer/#scoped-styling